src/main/java/seedu/parser/Parser.java

```diff
@@ -367,7 +392,7 @@ private static LocalDate getEventDate(String argument) throws EscException {
         DateTimeFormatter dateKey = DateTimeFormatter.ofPattern("[dd/MM/yyyy][d/M/yyyy][dd/MM/yy][d/M/yy]"
                 + "[yyyy/MM/dd][yyyy-MM-dd][yyyy-M-d]"
                 + "[dd-MM-yyyy][d-M-yyyy][dd-MM-yy][d-M-yy]"
-                + "[dd.MM.yy][d.M.yy][dd.MM.yyyy][d.M.yyyy][yyyy.MM.dd]"
+                + "[dd.MM.yy][d.M.yy][dd.MM.yyyy][d.M.yyyy]"
```

Comment:
Is the yyyy.MM.dd date format not working? That's why it's removed?

@JeremiasLiew please do take note and update the code/UG/DG as necessary

Reply:
I didn't change this part, should be someone else